dist.sh

```diff
@@ -60,3 +73,14 @@ done
 for f in $core_files; do
     cp $f $dist_dir/
 done
+
+case $platform in
+    windows)
+        rm -f $dist_dir/workcraft
+        7z a -r ${dist_dir}.zip $dist_dir >/dev/null
+        ;;
+    linux)
+        rm -f $dist_dir/workcraft.bat
+        tar -czf ${dist_dir}.tar.gz $dist_dir
```

Comment:
Another way is to have `dist_dir=dist` and `platform_dir="workcraft-$(git describe --tags)-$platform"` and glue them together only when necessary. I personally would go this route...